src/stores/rootStore.ts

```diff
@@ -148,7 +148,8 @@ export const useRootStore = defineStore('root', {
         (
           this.getBootstrapFilingStatus === FilingStatus.CHANGE_REQUESTED ||
           this.getBootstrapFilingStatus === FilingStatus.DRAFT ||
-          this.getBootstrapFilingStatus === FilingStatus.PENDING
+          this.getBootstrapFilingStatus === FilingStatus.PENDING ||
```

Comment:
By the way, Pending is after the user has filed the application but before payment is confirmed. In this case, we're well past the authorization stage, so your code in enum-utilities is good, but it's still in the Todo List until the user fixes the payment issue.